app/common/modules/led_rollover/led_rollover.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2025 PHYTEC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/led.h>
+#include <zephyr/sys/printk.h>
+
+#define LED_DEVICE_NODE DT_PATH(leds)
+
+/* Thread configuration */
+#define LED_ROLLOVER_STACKSIZE   1024
+#define LED_ROLLOVER_PRIORITY    5
+
+void led_rollover(void)
+{
+	const struct device *led_dev = DEVICE_DT_GET(LED_DEVICE_NODE);
+
+	if (!device_is_ready(led_dev)) {
+		printk("LED device not ready\n");
+		return;
+	}
+
+	printk("Starting LED blink demo using LED subsystem\n");
+
+	while (1) {
+		// Toggle each of the 4 LEDs on and off in sequence
+		for (int i = 0; i < 4; i++) {
+			led_on(led_dev, i);
+			k_msleep(200);
```

Comment:
Use k_sleep instead of k_msleep.
```suggestion
			k_sleep(K_MSEC(200));
```